static_report/src/components/shared/Footer.tsx

```diff
@@ -0,0 +1,49 @@
+import { Box, Flex, Text, Link, Image, Divider } from '@chakra-ui/react';
+
+export function Footer() {
+  return (
+    <Flex gap={8} my={8}>
+      <Flex gap={3}>
+        <Flex alignItems="center" gap={3}>
+          <Link href="https://github.com/InfuseAI/piperider" isExternal>
+            <Image
+              src="GitHub/GitHub-Logo.png"
+              width={5}
+              height={5}
+              alt="GitHub Logo"
+            />
+          </Link>
+          <Link
```

Comment:
Why two `<Link></Link>` instead of one for the entire Image + text?

Reply:
I checked the ticket, seems the logo didn't need to add links, let me remove it from the logo.